### PR TITLE
Fixes #6491

### LINKED ISSUE #6491: Janicart Bug
(WEB REPORT BY: orntoo REMOTE: 158.69.55.2:7777)
>Revision (Should be above if you're viewing this from ingame!)
82da52e0443da19444e800b56d7bd390819c6444 

>General description of the issue
Putting 2 bags on a janicart is bugged.

>What you expected to happen
Either it wouldn't let me put a second one on the cart or both would be hooked on.

>What actually happened
Had a bag on my cart, put a second one from maintenance on it and the first one disappeared completely along with everything inside it.

>Steps to reproduce if possible
Put 2 bags on cart.

### PR DESCRIPTION
**Pull request by Wizardcrying**
Fixes being able to destroy a trashbag on a janicart by putting on another one.

### CODE REVIEW

File: code/game/objects/structures/janicart.dm [hunk 1]
```diff
@@ -50,6 +50,9 @@
 		name = "upgraded janicart"
 		icon_state = "pussywagon_upgraded"
 	else if(istype(W, /obj/item/weapon/storage/bag/trash))
+		if(mybag)
+			user << "<span class='warning'>There is already a trashbag on the pimpin' ride!</span>"
```
Comment (PJB3005) [inline code/game/objects/structures/janicart.dm]: Use the `nick` var.

Comment (Wizardcrying) [inline code/game/objects/structures/janicart.dm]: ?

**Comment by Wizardcrying** [comment]
going to do a changelog thing

**Comment by dylanstrategie** [comment]
Please use the following

>\the [nick]
>(not the) [nick.name]

**Comment by Wizardcrying** [comment]
@dylanstategie what do you mean

**Comment by dylanstrategie** [comment]
When you cast an atom directly, it automatically adds "the" if it's lower case

Thus, you need to add the "\the" macro behind it so that it behaves as intended

If you do NOT want "the" to show up behind the atom's name (for example if it says "This atom" or "an atom"), you need to cast the atom's name so that it doesn't try to add "the"

**Comment by Wizardcrying** [comment]
Succeeded by  #6506 which got merged just as it was created.

**Pull request closed without merging**